recipes-browser/wpewebkit/wpewebkit.inc

```diff
@@ -13,7 +13,7 @@ DEPENDS = " \
 "
 
 inherit cmake pkgconfig perlnative python3native
-inherit ${@'cmake_qt5' if 'qt5-layer' in d.getVar('BBFILE_COLLECTIONS').split() else ''}
+inherit ${@bb.utils.contains('PACKAGECONFIG', 'qtwpe', 'cmake_qt5' if 'qt5-layer' in d.getVar('BBFILE_COLLECTIONS').split() else '', '', d)}
```

Comment:
Duno if wpe can be built with the `qtwpe` activated if the qt5-layer is not an available layer. If that is the case. Maybe it could be better implement this as a extra check step: 

```
python do_check_qtwpe () {
      if bb.utils.contains('PACKAGECONFIG', 'qtwpe', True, False, d) and not 'qt5-layer' in d.getVar('BBFILE_COLLECTIONS').split():
            bb.fatal("wpewebkit using 'qtwpe' requires 'qt5-layer'")
}

addtask check_qtwpe before do_fetch

inherit ${@bb.utils.contains('PACKAGECONFIG', 'qtwpe', 'cmake_qt5')}
```

Reply:
Sounds like a good idea now that you mention it. I will update the patch.

Reply:
yes. You are right. 

WDYT about this?

```
inherit ${@bb.utils.contains('PACKAGECONFIG', 'qtwpe', 'cmake_qt5')}

# qtwpe is set to ON if qt5-layer is available
PACKAGECONFIG_append = ${@'qtwpe' if 'qt5-layer' in d.getVar('BBFILE_COLLECTIONS').split() else ''}
```

Reply:
Actually this `inherit`line comes from @philn . Maybe he can give his opinion in this topic.